doc-gen/Makefile

```diff
@@ -0,0 +1,37 @@
+
+DEVICES = \
+    pca10040 \
+    microbit \
+    nrf52840-mdk \
+    pca10031 \
+    bluepill \
+    arduino \
+    digispark \
+    reelboard \
+    pca10056 \
+    arduino-nano33 \
+    itsybitsy-m0 \
+    feather-m0 \
+    feather-m4 \
+    trinket-m0 \
+    metro-m4-airlift \
+    circuitplay-express \
+    stm32f4disco \
+    circuitplay-express \
+    gameboy-advance \
+    itsybitsy-m4 \
+    nucleo-f103rb \
+    hifive1b \
+    pinetime-devkit0 \
+    x9pro
+
+all:
+	go run . $(DEVICES)
+
+
+#all:
+#	go run . pca10040 > ../content/doc/pca10040/machine.md
+#	go run . hifive1b > ../content/doc/hifive1b/machine.md
+#	go run . bluepill > ../content/doc/bluepill/machine.md
+#	go run . gameboy-advance > ../content/doc/gameboy-advance/machine.md
+#
```

Comment:
Oops looks like this is some old code that should really be removed.

Reply:
Yeah, I was wondering. :scissors: